src/lib/eina/eina_inline_private.h

```diff
@@ -20,8 +20,9 @@
 # define EINA_INLINE_PRIVATE_H_
 
 # include <time.h>
-#ifndef _MSC_VER
-# include <sys/time.h>
```

Comment:
What makes `sys/time` not needed?

Reply:
sys / time already comes from the branch  devs/joaoantoniocardoso/native-rebased/ecore

Reply:
By doing this, we are removing `sys/time` from Linux build. Is it intended?